src/main/resources/morph-hbz01-to-lobid.xml

```diff
@@ -16,6 +16,106 @@
 		<var name="ns-bibo" value="http://purl.org/ontology/bibo/"/>
 	</vars>
 	<macros>
+		<macro name="setRdfSubjectDescribedBy">
+			<combine name="~rdf:subject" value="${a}">
+				<data source="$[field]"/>
+				<data source="@id" name="a">
+					<regexp match="(.*)" format="$[ns-lobid-resource]${1}"/>
+				</data>
+			</combine>
+		</macro>
+		<macro name="setProvenance">
+			<combine name="$[field_1]" value="$[ns-lobid-organisation]${a}#!" flushWith="$[field_2]">
+				<choose name="a" flushWith="$[field_2]">
+					<data source="$[field_2]" name="a">
+						<lookup in="sigel2isilMap"/>
+					</data>
+					<data source="$[field_2]" name="a">
+						<!-- Off = Offline -->
+						<regexp match="[hH][bB][zZ]/[oO][fF][fF]" format="DE-605"/>
+					</data>
+					<data source="$[field_2]" name="a">
+						<!-- OZK = Online-Zentralkatalog" -->
+						<regexp match="HBZ/OZK" format="DE-605"/>
+					</data>
+					<data source="$[field_2]" name="a">
+						<regexp match="[hH][bB][zZ]/[rR][eE][dD]" format="DE-605"/>
+					</data>
+					<data source="$[field_2]" name="a">
+						<regexp match="HBZ/NWBibAlt" format="DE-605"/>
+					</data>
+					<data source="$[field_2]" name="a">
+						<regexp match="HBZ/NWBibOff" format="DE-605"/>
+					</data>
+					<data source="$[field_2]" name="a">
+						<regexp match="HBZ/NWBib" format="DE-605"/>
+					</data>
+					<data source="$[field_2]" name="a">
+						<regexp match="^NRW/Hist.Buch$" format="DE-605"/>
+					</data>
+					<data source="$[field_2]" name="a">
+						<regexp match="^NRW$" format="DE-605"/>
+					</data>
```

Comment:
There are several other resources with "^NRW" which probably are not from the HBZ. So I simply combined the definition to `^NRW$|^NRW/Hist.Buch$`.